pkgs/by-name/fl/flashlabel-a80/package.nix

```diff
@@ -0,0 +1,80 @@
+{
+  autoPatchelfHook,
+  cups,
+  fetchurl,
+  lib,
+  stdenv,
+}:
+
+stdenv.mkDerivation {
+  pname = "flashlabel-yxwl";
+  version = "1.2.1";
+
+  src = fetchurl {
+    url = "https://help.flashlabel.com/helpdesk/attachments/150080666194";
+    hash = "sha256-qkc3NJ1dK0nJf+Q7xL7f1/+X0COWSWMEbH4luzaFARc=";
+  };
+
+  # The driver is distributed as a self-extracting executable consisting of a
+  # shell script concatenated with a gzipped tar archive. The script hard codes
+  # its length in the `lines` variable, which we read to locate the archive.
+  unpackPhase = ''
+    lines="$(sed 's/^lines=//; t; d' "$src")"
+    tail --lines "+$lines" "$src" | tar --extract --gzip --strip-components=1
+  '';
+
+  patchPhase = ''
+    runHook prePatch
+
+    # Remove model from manufacturer name
+    sed --in-place 's/\(^\*Manufacturer: "YXWL\) [^"]*\("$\)/\1\2/' *.ppd
+
+    runHook postPatch
+  '';
+
+  nativeBuildInputs = [ autoPatchelfHook ];
+  buildInputs = [ cups ];
+
+  dontConfigure = true;
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir --parents $out/lib/cups/filter
+    mv {A80,A80H} $out/lib/cups/filter
```

Comment:
Are these the only filters?

Reply:
Yes. I like the idea of adding an assertion that no remain after the move but wasn’t sure if that’s common practice.